contrib/implements/README.md

```diff
@@ -0,0 +1,49 @@
+
+# The 'implements' tool
+
+implements is a small-ish tool created to compare the Ceph C APIs with
+go-ceph implmeents.
+
+## Build
+
+In the go-ceph repository run `make implmeents` to create a standalone
+binary for the `implements` cli tool.
+
+## Run
+
+```
+./implements [--verbose] [--json] [--list] [--annotate] <DIR> [pkg...]
```

Comment:
Could this be changed to have the same format as `go build` or `go test`? No need to be fancy, but my initial executions before reading this more carefully I only got empty output (`./implements ./rbd` and such).

Reply:
You just mean that you expect it to work with a ./ prefix on the packages, right? If so, sure.

Reply:
actually without the `<DIR>`, and only the package name, either local `./` prefixed or full-qualified-package-name `github.com/ceph/go-ceph/rbd`

Reply:
The former is done. Absolute paths work too.